### DISCUSSION TITLE
Suggestion:  Link on root Thingino page for Wyze V3 (and others)...

### DISCUSSION THREAD

**Discussion by HeyItsDaddy** [themactep/thingino-firmware#257] [Ideas]
I was scratching my head on how to find out which of the 4 SoC/WiFi combinations my 6 Wyze Cam V3's might fall under.  Took me a while to eventually find this page:
  https://github.com/wltechblog/thingino-installers/tree/main/wyze-cam-3
Which details the steps you take to find out exactly that information.  I wonder if linking to that from the "Wyze Cam V3" header on the root page would be beneficial to anyone else that's as dense as me.  :-P
![Thingino-WyzeV3](https://github.com/user-attachments/assets/4a20aca5-b4be-4c26-ae5c-1f436d695c21)

Second...for those of us that have already been using the wz_mini_hacks for forever, and have SSH access already to our cameras, is there something we can easily execute to get that information directly from the camera?  Just want to get an inventory of all my cameras before I even start, and that'd make it much faster/easier.

Thanks!!!

**Comment by HeyItsDaddy** [comment]
Answering my second question myself:  I copied the files soc and thinginoflash from the wyzev3-flasher.zip to my own /wz_mini/bin folder, then just ran the thinginoflash script on SSH and it created the /opt/flash.txt file I was looking for.  :-P

**Comment by virmaior** [comment]
haven't made the switch yet but did the same. It'd be nice if wltechblog had posted that code and not just the version that already has it integrated.

**Reply by gtxaspec** [reply]
i'll write up a wiki article on how to do this.

**Comment by gtxaspec** [comment]
If you need to determine the SoC type, you can run this command on the device, provided you have shell access to the device:

```curl -s -L https://github.com/themactep/thingino-firmware/raw/refs/heads/master/overlay/lower/usr/sbin/soc | sh -s - -m```

this will download and run thingino's soc script which will tell you the exact model of SoC you have.

**Reply by virmaior** [reply]
by the way, running that code on my Atom Cam 2 shows "t31zx" but thingino.com lists atom cam2 as t31x

**Reply by gtxaspec** [reply]
thats OK, as z/zx are interchangeable
